packages/web-components/package.json

```diff
@@ -47,19 +49,20 @@
     "test-firefox:verbose": "karma start karma.conf.js --browsers=FirefoxHeadless --single-run --coverage --reporter=mocha"
   },
   "devDependencies": {
-    "@microsoft/eslint-config-fast-dna": "^1.2.0",
     "@microsoft/api-extractor": "7.7.1",
-    "@storybook/addons": "^5.3.8",
-    "@storybook/cli": "^5.3.8",
-    "@storybook/html": "^5.3.8",
-    "@storybook/theming": "^5.3.8",
+    "@storybook/addons": "^6.1.17",
```

Comment:
Can you try removing all the separate storybook deps? It's probably best if you can use the same version as the rest of the repo, unless there's a compelling reason not to. (@TristanWatanabe is currently working on upgrading us to 6.2.)

Reply:
I've removed those which are consistent - without knowing the implications I'm hesitant to hoist theming and html to the root as they are not used elsewhere from what I can tell.